htdocs/accountancy/journal/bankjournal.php

```diff
@@ -432,6 +432,7 @@
 					$bookkeeping->journal_label = $journal_label;
 					$bookkeeping->fk_user_author = $user->id;
 					$bookkeeping->date_create = $now;
+					$bookkeeping->date_lim_reglement = $now;
```

Comment:
$now ? This should come from field date_lim_reglement from invoice ?

Reply:
For a bank journal into bookkeeping, we cannot set date_lim_reglement, because on bank transaction can covert more than one invoice to pay or to gain. Should we put the date of bank transaction instead ?

Reply:
Having a field with content that can have several values means this field is probably useless into the bookkeeping table.
@Darkjeff Why do we have such a field into table bookkeeping ? It is a field related to billing so to invoice and not to accountancy report.

Reply:
It's for export accountancy : sage and FEC

Reply:
It seems this field is not commented to be a FEC field. I don't think FEC ask a field with a content that has no sense.
So when you have a paiement on 2 invoices, once on invoice with a limit date 2019-01-01 and the other invoice with limite date 2019-02-02, which value the FEC standard ask to have ?
 2019-01-01 or  2019-02-02 ?

Reply:
this field "date_lim_reglement" is just for sales journal and purchase journal, we don't need this information for bankjournal. 
And not FSC (sorry), but sage and quadratus

Reply:
So @atm-florian I suggest to set value to "null" instead of now for line 435.